core/templates/dev/head/domain/skill/SkillRightsObjectFactory.ts

```diff
@@ -16,7 +16,10 @@
  * @fileoverview Factory for creating and mutating instances of frontend
  * skill rights domain objects.
  */
-
+export interface ISkillRightBackendInterface {
```

Comment:
I don't think Interface word is required since it is starting with the capital I.
So ISkillRightBackendInterface -> ISkillRight

Reply:
Changed it to ISkillRight